codespell_lib/data/dictionary.txt

```diff
@@ -2248,6 +2252,7 @@ deatches->detaches
 deatching->detaching
 debain->Debian
 debateable->debatable
+debiab->Debian
 Debiab->Debian
```

Comment:
`debiab` already exists in the dictionary

Reply:
I didn't realize that you weren't managing case